opm/autodiff/BlackoilAquiferModel_impl.hpp

```diff
@@ -0,0 +1,138 @@
+namespace Opm {
+
+
+    template<typename TypeTag>
+    BlackoilAquiferModel<TypeTag>::
+    BlackoilAquiferModel(Simulator& ebosSimulator,
+                      const ModelParameters& param,
+                      const bool terminal_output)
+        : ebosSimulator_(ebosSimulator)
+        , param_(param)
+        , terminal_output_(terminal_output)
+    {
+        // const auto& gridView = ebosSimulator_.gridView();
+
+        // number_of_cells_ = gridView.size(/*codim=*/0);
+        // global_nc_ = gridView.comm().sum(number_of_cells_);
+        gravity_ = ebosSimulator_.problem().gravity()[2];

```

Comment:
Since ebosSimulator_ is a member, gravity_ does not need to be a member variable here.